handlers/syslog_test.go

```diff
@@ -28,11 +28,13 @@ func (m *mockProducer) Close() {
 
 func setup(t *testing.T) (*echo.Echo, func()) {
 	e := echo.New()
-	handler, err := NewSyslogHandler("t0ken", &mockProducer{t: t})
-
+	syslogHandler, err := NewSyslogHandler("t0ken", &mockProducer{t: t})
 	assert.Nilf(t, err, "Expected NewSyslogHandler() to succeed")
+	ironHandler, err := NewIronIOHandler("t0ken", &mockProducer{t: t})
```

Comment:
Can this be made conditional using configuration/command line parameters to reduce surface area?